docs-devsite/auth.recaptchaconfig.md

```diff
@@ -0,0 +1,35 @@
+Project: /docs/reference/js/_project.yaml
```

Comment:
This file should be removed.

Reply:
Not sure what's this used for but it's generated in a lot of other places.